content/guides/core-concepts/cypress-app.md

```diff
@@ -14,10 +14,90 @@ title: Cypress App
 
 ## Overview
 
-Cypress runs tests in a unique interactive application that allows you to see
-commands as they execute while also viewing the Application or Component Under
-Test.
+The Cypress App is a desktop application with powerful features for creating,
+configuring, browsing and most importantly running your tests. We'll touch on a
+few of the different functions before moving on to the Test Runner, the heart of
+the Cypress testing experience.
 
+<Alert type="warning">
+
+<strong class="alert-header">Dashboard Syndication</strong>
+
+A number of the features described below will only be shown if you are logged
```

Comment:
I wouldn't want to be too sales-y, but perhaps we can add a descriptive statement here for why it's valuable for users to have this data here, e.g.,

```suggestion
As the Cypress app runs tests, it records result data to the Cypress Dashboard. Dashboard derives insights to help you measure and optimize the entire development process. 
```

Reply:
I've added something in my latest changes and put this on the to-do list for resolving.